test/jdk/javax/swing/JMenuBar/TestMenuMnemonic.java

```diff
@@ -31,69 +31,52 @@
  * @run main TestMenuMnemonic
  */
 
-import java.awt.event.KeyEvent;
 import java.awt.Robot;
+import java.awt.event.KeyEvent;
 import java.util.concurrent.atomic.AtomicInteger;
 import javax.swing.JFrame;
 import javax.swing.JMenu;
-import javax.swing.JMenuItem;
 import javax.swing.JMenuBar;
+import javax.swing.JMenuItem;
 import javax.swing.MenuElement;
 import javax.swing.MenuSelectionManager;
 import javax.swing.SwingUtilities;
 import javax.swing.UIManager;
+
 import com.sun.java.swing.plaf.windows.WindowsLookAndFeel;
 
 public class TestMenuMnemonic {
 
     private static JFrame frame;
     private static JMenuBar menuBar;
     private static JMenu fileMenu;
+    private static final AtomicInteger mnemonicHideCount = new AtomicInteger(0);
+    private static final AtomicInteger mnemonicShowCount = new AtomicInteger(0);
 
     public static void main(String[] args) throws Exception {
         UIManager.setLookAndFeel("com.sun.java.swing.plaf.windows.WindowsLookAndFeel");
-        int expectedMnemonicShowHideCount = 5;
+        final int EXPECTED = 5;
```

Comment:
I meant making it `private static final` outside of main.

I suggest something like this:

```java
    private static JFrame frame;
    private static JMenuBar menuBar;
    private static JMenu fileMenu;

    private static final AtomicInteger mnemonicHideCount = new AtomicInteger(0);
    private static final AtomicInteger mnemonicShowCount = new AtomicInteger(0);

    private static final int EXPECTED = 5;
```

The blank lines separate the blocks of related fields.

Reply:
>private static final int EXPECTED = 5;
this var is sued only in mail, so I kept it as local variable.

Reply:
There's a difference between _a local variable_ that it is now and _a constant_ declared at the class level. In the latter case, the compiler may eliminate the constant and inline its value; in the former case, it can't do it.

I am still for making it a real constant declared at the class level as per my suggestion above.

Reply:
Updated.